source/MapPlanetCard.cpp

```diff
@@ -287,8 +287,8 @@ void MapPlanetCard::Highlight(double availableSpace) const
 	const Interface *planetCardInterface = GameData::Interfaces().Get("map planet card");
 	const double width = planetCardInterface->GetValue("width");
 
-	FillShader::Fill(Point(Screen::Left() + width / 2., yCoordinate + availableSpace / 2.),
-		Point(width, availableSpace), *GameData::Colors().Get("item selected"));
+	FillShader::Fill(Rectangle::FromCorner(Point(Screen::Left(), yCoordinate),
+		Point(width, availableSpace)), *GameData::Colors().Get("item selected"));
```

Comment:
I think two expressions is better than one expression that is split across two lines.
```suggestion
	Rectangle highlightRegion = Rectangle::FromCorner(Point(Screen::Left(), yCoordinate), Point(width, availableSpace));
	FillShader::Fill(highlight, *GameData::Colors().Get("item selected"));
```

Reply:
I don't like that the expression is split. I'm not going to prefer splitting it somewhere else to using two expressions.

Reply:
> I don't like that the expression is split. I'm not going to prefer splitting it somewhere else to using two expressions.

@warp-core what are you asking for, then?  I think Tom's guessed at a few different things already...  The line is 157 char wide -- it **has** to be split -- or the variable names shortened, but c'mon: that can't be the answer.